config/hubs/schema.yaml

```diff
@@ -74,6 +75,38 @@ properties:
       - project
       - cluster
       - zone
+  aws:
+    type: object
+    additionalProperties: false
+    description: |
+      Configuration to connect to the cluster on AWS. Is used when
+      provider is set to `aws`
+    properties:
+      key:
+        type: string
+        description: |
+          Path to a `sops` encrypted JSON key that contains AWS credentials
+          with enough rights to get full access to the kubernetes cluster.
+      clusterType:
+        type: string
```

Comment:
Is this acceptable in the schema of JSONschema file when we also have an `enum` as part of a JSON schema file restricting it to two specific string values already?

Reply:
I followed the pattern I saw in the rest of the schema file and it seems to be valid and acceptable, AFAIK.